### PR TITLE
chore: remove github-bot from codeowners

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Remove `github-bot` from codeowners.

### CODE REVIEW

**Comment by derberg** [comment]
Context for those that will approve it:
- github after 3 years of Codeowners feature in production started validating the files
- current codeowners setup is broken, it is not possible to add user with `[]` in name 

We now remove the `github-actions[bot]`. And then we will test if even though we have branch protection that requires CODEOWNERS approval, `github-actions[bot]` approval will be accepted for merge too

**Review by mcturco — APPROVED** [review]

**Pull request merged**
